package.json

```diff
@@ -5,11 +5,10 @@
   "version": "0.1.0",
   "private": true,
   "engines": {
-    "node": ">=20",
-    "pnpm": "8"
+    "node": ">=20"
```

Comment:
we should add in the pnpm version number here like before. it can just be updated to 9